client/js/App.js

```diff
@@ -1,10 +1,18 @@
 import React, {Component} from 'react';
+import {Platform, UIManager} from 'react-native';
 import {ApolloProvider} from 'react-apollo';
 import client from './config/api';
-
 import RootStackNavigator from '../js/navigation/RootStackNavigator';
 
 export default class App extends Component {
+  componentDidMount() {
+    if (Platform.OS === 'android') {
+      if (UIManager.setLayoutAnimationEnabledExperimental) {
+        UIManager.setLayoutAnimationEnabledExperimental(true);
+      }
+    }
+  }
```

Comment:
Nice work but remember not to spend too much time on making features compatible for android. Our focus for this project is iOS only.

Reply:
removed